src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -471,20 +471,26 @@ def _remove_leading_invalid_chars(self, text: str):
     def _preprocess_response(
         self, res_timestamps: list[int], res_outputs: list[dict[str, str]]
     ) -> None:
+        """Helper function to preprocess responses of a request."""
         # FIXME -- remove this triton code once it is properly fixed in PA
         # (PA/triton will add junk to the start of the BYTES array. Remove it here)
         if self._service_kind == "triton":
             for d in res_outputs:
                 d["text_output"] = self._remove_leading_invalid_chars(d["text_output"])
+        elif self._service_kind == "openai":
+            # remove the null final response in streaming mode
+            response = res_outputs[-1]["response"]
```

Comment:
can you rename the variable to last_response?

Reply:
other than this, LGTM

Reply:
Changed the var name